frontend/components.js

```diff
@@ -703,7 +704,7 @@ const LoginStatus = connect(
         {class: 'inline', for: 'navcollapse'},
         h(
           'div',
-          {class: 'button', onClick: logout},
+          {class: 'button', onClick: () => setTimeout(logout, 100)},
```

Comment:
Why is needed this change?

Reply:
to fix #102, onclick action bypasses other (html: label for checkbox to show/hide mobile menu) + adds time for amination